### DISCUSSION TITLE
How can I quickly add a "new line(\n)" at the end of the default theme's prompt?

### DISCUSSION THREAD

**Discussion by denik1981** [JanDeDobbeleer/oh-my-posh#1897] [Troubleshoot] [answered]
I have started with OhMyPosh just yesterday. Sorry about being lazy on the docs. I just want to override the default theme by only adding a new line at the end of the prompt starting at the second line leaving me more room to type without the need of hitting `\\`. I prefer not to replace it with the transient prompt.  

BTW, I'm in Linux using zsh

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can simply add a `\n` to the last segment or create a new text segment with just that content if the template.

**Reply by denik1981** [reply]
Thanks for the tip and your contribution as well!

**Comment by JoshPaulie** [comment]
Here's an example (toml) snippet for those googling this in the feature. (Arrow optional ofc)

```toml
  [[blocks.segments]]
  type = "text"
  style = "plain"
  template = "\n ↳"
```

**Reply by nachocab** [reply]
It works with the arrow:

![image](https://github.com/user-attachments/assets/8c15ec2e-ce06-471f-aae4-9927731a5d25)

But if I remove it, the newline is removed:

![image](https://github.com/user-attachments/assets/62ac1004-bc82-4a7c-89e6-a1f872b3dbb9)

![image](https://github.com/user-attachments/assets/836d0dd1-9dea-4c6c-bcf5-2e7994a2c57a)


I'd like to avoid this because if I triple click a command, the initial character is also copied:

![image](https://github.com/user-attachments/assets/0e972995-3e4e-4cca-b17d-2240b6ac1f5a)

**Reply by JanDeDobbeleer** [reply]
Add `force: true` to the block

**Comment by jasonm23** [comment]
to force just a newline you have to use `" \n"`
